lib/controllers/embed.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+var winston = module.parent.require('winston');
+
+var nbbAuthController = require.main.require('./src/controllers/authentication');
```

Comment:
Arrange these imports in a sensible manner. Either plugin's own imports first or parent imports first.

Reply:
done